dockerfiles/compilation/java-6/compile.sh

```diff
@@ -43,14 +43,26 @@ git checkout $SHA1
 
 # Compilation
 
+## Compile and install maven mailetdocs plugin
+mvn clean package install:install -pl mailet/mailetdocs-maven-plugin
+
+## Compile James
 if [ "$SKIPTESTS" = "skipTests" ]; then
    mvn package -DskipTests -Pjpa
 else
    mvn package -Pjpa
 fi
 
-# Retrieve result
-
 if [ $? -eq 0 ]; then
    cp server/app/target/james-server-app-*-app.zip $DESTINATION
 fi
+
+## Generate the mailetdocs
+mvn org.apache.james:mailetdocs-maven-plugin:2.5.1-SNAPSHOT:aggregate
```

Comment:
The point is when we will modify this mojo, we will not be able to test that it's working on Jenkins.
We should release, upload; if Jenkins fails due to error, then we should re-release; re-upload.